docs/source/guide/export.md

```diff
@@ -107,6 +107,33 @@ For a large labeling project with hundreds of thousands of tasks, do the followi
 
 Label Studio supports many common and standard formats for exporting completed labeling tasks. If you don't see a format that works for you, you can contribute one. For more information, see the [GitHub repository for the Label Studio Converter tool](https://github.com/HumanSignal/label-studio-converter).
 
+
+#### Text, audio, and video  tags
+
+| Export Format | Text | Audio | Labels | Choices | TextArea | TimeSeries | Video | Comments |
+|---------------|------|-------|--------|---------|----------|------------|-------|----------|
+| **ASR_MANIFEST** | ❌ | ✅ | ❌ | ❌ | ✅ | ❌ | ❌ | Audio + TextArea only |
+| **CoNLL2003** | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | Text + Labels only |
+| **CSV** | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | All project types |
+| **JSON** | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | All project types |
+| **JSON_MIN** | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | All project types |
+| **spaCy** | ✅ | ❌ | ✅ | ❌ | ❌ | ❌ | ❌ | Text + Labels (via CoNLL2003) |
+| **TSV** | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | ✅ | All project types |
+
+#### Image segmentation tags
+
+| Export Format | BrushLabels | RectangleLabels | KeyPointLabels | PolygonLabels | Comments |
+|---------------|-------------|-----------------|----------------|---------------|----------|
+| **NumPy** | ✅ | ❌ | ❌ | ❌ |  |
+| **COCO** | ✅ | ✅ | ✅ | ✅ | See [note](#kpl-note) on KeyPointLabels |
+| **CSV** | ✅ | ✅ | ✅ | ✅ | All project types |
+| **JSON** | ✅ | ✅ | ✅ | ✅ | All project types |
+| **JSON_MIN** | ✅ | ✅ | ✅ | ✅ | All project types |
+| **Pascal VOC XML** | ❌ | ✅ | ❌ | ❌ | RectangleLabels only |
+| **TSV** | ✅ | ✅ | ✅ | ✅ | All project types |
+| **YOLO** | ❌ | ✅ | ✅ | ✅ | See [note](#kpl-note) on KeyPointLabels |
```

Comment:
please check the text where this link points;
<img width="1934" height="1223" alt="image" src="https://github.com/user-attachments/assets/97985982-232e-42c6-9116-7e0162be44c7" />